README.md

```diff
@@ -1,22 +1,122 @@
 <div align='center'>
 <h1>MiV - Simulator</h1>
-
-[![Documentation Status][badge-documentation]][link-documentation]
 </div>
 
 Bio-physical neural network simulator for Mind-in-Vitro in context of processing and computation.
 
-## Documentation
+## Generic installation
```

Comment:
We moved the installation and user guidelines into the documentation. It looks like the file is copied from the previous branch. Can we revert the change?

Reply:
Yes, absolutely